### DISCUSSION TITLE
Can I wrap blocks inside custom tool?

### DISCUSSION THREAD

**Discussion by ggukggukee** [codex-team/editor.js#2362] [Q&A]
Can I wrap blocks inside custom tool?
for example: I want wrap blocks inside div that has border.

**Comment by neSpecc** [comment]
Yes, you can create a Block Tune for that using the [Block Tunes API](https://editorjs.io/block-tunes-api/)

**Reply by ggukggukee** [reply]
Is there any plugin that was created using Block Tunes API, so I can see how it's done?

**Reply by neSpecc** [reply]
> Is there any plugin that was created using Block Tunes API, so I can see how it's done?

https://github.com/editor-js/text-variant-tune

**Reply by ggukggukee** [reply]
Can I use Block Tunes API to wrap multiple blocks?
